lib/index.js

```diff
@@ -75,6 +76,7 @@ module.exports = function ({
   },
   useMasterAccessToken = false,
   sessionKey = 'session',
+  sessionAuthorizationKeySecret=undefined,
```

Comment:
Would be more consistent to leave unassigned. I didn't assign anything to `customize` below, as it seems unnecessary. Side comment: this would give you "error  Operator '=' must be spaced" if you lint. So maybe:
```
  authorizationId,
```

Reply:
agree, this would be more consistent. I will adopt this